app.py

```diff
@@ -16,13 +16,18 @@ def chat():
     inquiry = request.args.get('inquiry', '')
     print(f"    Human: {inquiry}")
 
-    def stream(part):
-        yield part
+    # Check for duplicate inquiry
+    if history and history[-1]['inquiry'] == inquiry:
+        answer = "Your inquiry is the same as before, please change the inquiry."
+    else:
+        def stream(part):
+            yield part
```

Comment:
kalau duplicated inquiry nya kenapa sat?

and this function kenapa didefine di if statement?

Reply:
kalo duplicate, assistant ga bakal response aka error, nah itu pake if else biar user ganti dulu ke inquiry yg lain, terus kalo mau balik lagi ke inquiry sebelumnya bisa response si assistant